<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InquireDialog</class>
 <widget class="QDialog" name="InquireDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBoxItem_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>125</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentText">
    <string>Book Title</string>
   </property>
   <item>
    <property name="text">
     <string>Book Title</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Author</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Publisher</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Publisher City</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Barcode</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditKey1_1">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>160</y>
     <width>136</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditKey2_1">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>160</y>
     <width>136</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAON_1">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>160</y>
     <width>56</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>And</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Or</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxAcc_1">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>160</y>
     <width>79</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Precise</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fuzzy</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>122</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Publication Date :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditFrom">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>136</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>From</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>40</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>To</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditTo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>512</x>
     <y>40</y>
     <width>136</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>122</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pages :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>62</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Between</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxLess">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>136</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>80</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>And</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxMore">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>512</x>
     <y>80</y>
     <width>136</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>122</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Other Options :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>120</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Query Conditon</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Plus Query Conditon</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxItem_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>125</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentText">
    <string>Book Title</string>
   </property>
   <item>
    <property name="text">
     <string>Book Title</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Author</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Publisher</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Publisher City</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Barcode</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxAON_2">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>200</y>
     <width>56</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>And</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Or</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditKey1_2">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>200</y>
     <width>136</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAcc_2">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>200</y>
     <width>79</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Precise</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fuzzy</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditKey2_2">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>200</y>
     <width>136</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditKey1_3">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>240</y>
     <width>136</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAcc_3">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>240</y>
     <width>79</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Precise</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fuzzy</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxItem_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>125</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentText">
    <string>Book Title</string>
   </property>
   <item>
    <property name="text">
     <string>Book Title</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Author</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Publisher</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Publisher City</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Barcode</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxAON_3">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>240</y>
     <width>56</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>And</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Or</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditKey2_3">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>240</y>
     <width>136</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditKey1_4">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>280</y>
     <width>136</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAcc_4">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>280</y>
     <width>79</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Precise</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fuzzy</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxAON_4">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>280</y>
     <width>56</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>And</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Or</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditKey2_4">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>280</y>
     <width>136</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxItem_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>125</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentText">
    <string>Book Title</string>
   </property>
   <item>
    <property name="text">
     <string>Book Title</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Author</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Publisher</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Publisher City</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Barcode</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditKey1_5">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>320</y>
     <width>136</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAcc_5">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>320</y>
     <width>79</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Precise</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fuzzy</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxAON_5">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>320</y>
     <width>56</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>And</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Or</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditKey2_5">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>320</y>
     <width>136</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxItem_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>125</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentText">
    <string>Book Title</string>
   </property>
   <item>
    <property name="text">
     <string>Book Title</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Author</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Publisher</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Publisher City</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Barcode</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxItemAON_2">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>200</y>
     <width>56</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>And</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Or</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxItemAON_3">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>240</y>
     <width>56</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>And</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Or</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxItemAON_4">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>280</y>
     <width>56</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>And</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Or</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxItemAON_5">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>320</y>
     <width>56</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>And</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Or</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonConfirm">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>380</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Inquire</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>380</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlus">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxPubdate">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>21</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxPages">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>21</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxItemAON_1">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>160</y>
     <width>56</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>And</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Or</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
